lib/google-hangouts-chat/setup.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const prompt = require('souffleur');
+const googleHangoutsParse = require('./parse');
+const googleHangoutsReply = require('./reply');
+const color = require('../console-colors');
+const envUtils = require('../utils/env-utils');
+
+module.exports = function googleHangoutsSetup(api, bot, logError, optionalParser, optionalResponder) {
+  let parser = optionalParser || googleHangoutsParse;
+  let responder = optionalResponder || googleHangoutsReply;
```

Comment:
We can use `const` instead of `let` for both `parser` and `responder`, as they are not re-assigned. I guess should update that for other bots too.